resources/pgp-keys-map.list

```diff
@@ -591,6 +591,8 @@ org.osgi:org.osgi.compendium    = noSig
 
 org.osgi                        = 0x1AF010D408C1852AB6EFD4B651CDB4A9DAAC1CBC
 
+org.ostermiller:utils:1.07.00   = noSig
```

Comment:
There is only this version
https://repo1.maven.org/maven2/org/ostermiller/utils/
so specific versions range here is not need now

Reply:
I have rebased and removed the version range.